doc/classes/EditorPlugin.xml

```diff
@@ -245,7 +245,7 @@
 			<return type="Texture2D">
 			</return>
 			<description>
-				Override this method in your plugin to return a [Texture2D] in order to give it an icon.
```

Comment:
If you want to remove this, then this should be rephrased because "to return a Texture2D to give it an icon" is not easy to understand.